### DISCUSSION TITLE
Welcome to yas Discussions!

### DISCUSSION THREAD

**Discussion by thiennn** [nashtech-garage/yas#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ngohang26** [comment]
có thể cho tôi biết các bạn sẽ mô hình này như thế nào và sử dụng phần mềm nào không
![Uploading image.png…]()
